### DISCUSSION TITLE
How to administer the selfhost version

### DISCUSSION THREAD

**Discussion by parzifox** [stoatchat/discussions#673] [Q&A]
After examining the repositories, I did not find any information and tools for administration. Therefore, the question arose how to administer?
PS. Sorry for the bad English.

**Comment by phuocthanhlamnguyen-gif** [comment]
Firstly, if the project is a binary, type `./program --admin --reset-config` or `./program --status`; otherwise, if it's to manage users, clean up logs, etc., then you can write a script in Python or Bash. Lastly, you can change the .json, .yml, etc., and restart the program, so that's it, if this guide is helptful, then please tell me =).

**Comment by IAmTomahawkx** [comment]
We do not bundle administrative tools. Any administration you wish to do must be done directly against the database.
